coalib/output/ConsoleInteraction.py

```diff
@@ -304,25 +305,28 @@ def print_results_formatted(log_printer,
 
     for result in result_list:
         severity_str = RESULT_SEVERITY.__str__(result.severity)
+        args = vars(result)
         try:
             if len(result.affected_code) == 0:
+                args['affected_code'] = None
                 print(format_str.format(file=None,
                                         line=None,
                                         end_line=None,
                                         column=None,
                                         end_column=None,
                                         severity_str=severity_str,
-                                        **result.__dict__))
+                                        **args))
                 continue
 
             for range in result.affected_code:
+                args['affected_code'] = range
```

Comment:
line number 321(which is already in master branch) looks wrong because somebody is assigning the value to the built-in function `range`. I'll raise an issue by suggesting changes for that to something like `sourcerange` instead.